client/Cargo.toml

```diff
@@ -18,7 +18,7 @@ name = "bitcoincore_rpc"
 path = "src/lib.rs"
 
 [dependencies]
-bitcoincore-rpc-json = "0.2"
+bitcoincore-rpc-json = { version = "0.2", path = "../json" }
```

Comment:
This doesn't work for releasing.

Reply:
It should. I use it all over the place. Eg. https://github.com/dpc/crev/blob/609888cedd9e9484d279113467a87d758470aa57/cargo-crev/Cargo.toml#L24

Reply:
So how do you release? Last time I tried to `cargo publish` a crate with a `path = ` dependency, I got a message telling me that path dependencies are not allowed or something.

Reply:
It just works. Always worked. :) You first release the `json`, then you go to the client and release that. `path` is ignored when doing `cargo publish`, AFAIK.